src/components/NewWorkspaceModal.ts

```diff
@@ -193,26 +221,30 @@ const NewWorkspaceModal = withDisplayName(
       return !!cloneWorkspace && bucketLocation !== sourceWorkspaceLocation;
     };
 
-    const isAzureBillingProject = (project) => isCloudProviderBillingProject(project, cloudProviders.azure.label);
+    const isAzureBillingProject = (project?: BillingProject): boolean =>
+      isCloudProviderBillingProject(project, 'AZURE');
```

Comment:
I removed `cloudProviders` here to avoid mixing Leo's cloud provider enums with the enums used for Billing.